jsonschema-core.xml

```diff
@@ -1121,6 +1143,31 @@
                         validation of core keywords.
                     </postamble>
                 </figure>
+                <t>
+                    "localKeyword" and "startDate" are now valid for any schema that
+                    declares this meta-schema for their "$schema".  Note that defining
+                    it in this way does not imply the new keywords are valid within an
+                    instance to be validated by such a schema.
+                </t>
+                <t>
+                    Although the meta-schema can validate the syntax of "startDate"
+                    within a schema, it is the vocabulary that defines the logic behind
+                    the semantic meaning of "startDate".  Without an understanding of
+                    the semantics (that the instance value should represent a date after that
+                    given in the schema), an implementation can only validate the syntactic
```

Comment:
> that the instance value should represent a date after that given in the schema

Would it be valid to add this in the comment for this field, showing that information is contained in annotation which isn't avilable in the meta-schema validation mode, which is required for "proper use" of the vocabulary?

Reply:
Good catch. I changed the example mid-way through. I need to update this note.

Reply:
Possibly, sure. But I don't know that it's necessary for this example.  I put it here to contrast the syntactic usage later in the sentence.